lib/ansible/module_utils/facts.py

```diff
@@ -1943,6 +1944,35 @@ def get_dmi_facts(self):
         rc, out, err = self.module.run_command("/usr/sbin/lsattr -El sys0 -a fwversion")
         data = out.split()
         self.facts['firmware_version'] = data[1].strip('IBM,')
+        rc, out, err = self.module.run_command("/usr/sbin/lsconf")
+        if out:
```

Comment:
i've made revision, add the get_bin_path and rc==0 as u metioned before, if there is anything left pls let me know